model/simulationtests/heatpump_plantloop_eir.rb

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+require 'openstudio'
+require 'lib/baseline_model'
+
+model = BaselineModel.new
+
+# make a 1 story, 100m X 50m, 1 zone building
+model.add_geometry({ 'length' => 100,
+                     'width' => 50,
+                     'num_floors' => 1,
+                     'floor_to_floor_height' => 4,
+                     'plenum_height' => 0,
+                     'perimeter_zone_depth' => 0 })
+
+# add windows at a 40% window-to-wall ratio
+model.add_windows({ 'wwr' => 0.4,
+                    'offset' => 1,
+                    'application_type' => 'Above Floor' })
+
+# add thermostats
+model.add_thermostats({ 'heating_setpoint' => 24,
+                        'cooling_setpoint' => 28 })
+
+# assign constructions from a local library to the walls/windows/etc. in the model
+model.set_constructions
+
+# set whole building space type; simplified 90.1-2004 Large Office Whole Building
+model.set_space_type
+
+# add design days to the model (Chicago)
+model.add_design_days
+
+# In order to produce more consistent results between different runs,
+# we sort the zones by names (only one here anyways...)
+zones = model.getThermalZones.sort_by { |z| z.name.to_s }
+zone = zones[0]
+
+# PlantLoopHeatPump_EIR_WaterSource.idf
+hw_loop = OpenStudio::Model::PlantLoop.new(model)
+hw_loop.setName('Hot Water Loop')
+hw_loop.setMinimumLoopTemperature(10)
+hw_temp_f = 140
+hw_delta_t_r = 20 # 20F delta-T
+hw_temp_c = OpenStudio.convert(hw_temp_f, 'F', 'C').get
+hw_delta_t_k = OpenStudio.convert(hw_delta_t_r, 'R', 'K').get
+hw_temp_sch = OpenStudio::Model::ScheduleRuleset.new(model)
+hw_temp_sch.defaultDaySchedule.addValue(OpenStudio::Time.new(0, 24, 0, 0), hw_temp_c)
+hw_stpt_manager = OpenStudio::Model::SetpointManagerScheduled.new(model, hw_temp_sch)
+hw_stpt_manager.addToNode(hw_loop.supplyOutletNode)
+sizing_plant = hw_loop.sizingPlant
+sizing_plant.setLoopType('Heating')
+sizing_plant.setDesignLoopExitTemperature(hw_temp_c)
+sizing_plant.setLoopDesignTemperatureDifference(hw_delta_t_k)
+# Pump
+hw_pump = OpenStudio::Model::PumpVariableSpeed.new(model)
+hw_pump_head_ft_h2o = 60.0
+hw_pump_head_press_pa = OpenStudio.convert(hw_pump_head_ft_h2o, 'ftH_{2}O', 'Pa').get
+hw_pump.setRatedPumpHead(hw_pump_head_press_pa)
+hw_pump.setPumpControlType('Intermittent')
+hw_pump.addToNode(hw_loop.supplyInletNode)
+
+cw_loop = OpenStudio::Model::PlantLoop.new(model)
```

Comment:
I'd move this loop either above or below the two other. perhaps make it into a block comment

```
############   C O N D E N S E R    S I D E  ##################
cw_loop = ...

############ HEATING / COOLING (LOAD) LOOPS  ############
hw_loop = ...

chw_loop = ...
```

Reply:
Also, from a cursory look, you don't have an equipment on the supply side of this condenser loop do you? @joseph-robertson